### PR TITLE
Completed the JavaScript Light Bulb exercise

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

File: javascript-light-bulb/styles.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.light-container {
+  margin: 0;
+  background-color: rgb(255, 254, 224);
+}
+
+.light {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (codingcodymiller) [inline javascript-light-bulb/styles.css:27]: Please go through all of your CSS and turn the properties off one at a time, and make sure that all of them are actually doing something.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Owen! One fix here:

**Comment by owen-adams2021** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Much better! Very nice 🎉 🎉 🎉

**Pull request merged**